test/extended/cli/explain.go

```diff
@@ -87,12 +86,12 @@ var (
 		{Group: "monitoring.coreos.com", Version: "v1", Resource: "servicemonitors"},
 
 		// FIXME
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "containerruntimeconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "controllerconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "kubeletconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigpools"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigs"},
-		// schema.GroupVersionResource{Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "mcoconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "containerruntimeconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "controllerconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "kubeletconfigs"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigpools"},
+		// {Group: "machineconfiguration.openshift.io", Version: "v1", Resource: "machineconfigs"},
```

Comment:
I'm not sure how we can move this type to openshift/api. machineconfigs depends on Ignition which we don't control so we can't add tags for autogenerated stuff. We're moving what we can from above in https://github.com/openshift/api/pull/301 but there's something which we need to understand and fix yet

Reply:
@runcom not everything needs to be in openshift/api to get it tested. The gist is to publish those information when creating those CRDs.